app/src/main/java/org/greenstand/android/TreeTracker/camera/ImageReviewScreen.kt

```diff
@@ -22,6 +22,7 @@ import androidx.compose.ui.layout.ContentScale
 import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.unit.dp
+import kotlinx.coroutines.launch
```

Comment:
Do we need this?

Reply:
Yes it's been used in the accept button to approve images in the view model.